core/src/main/java/org/elasticsearch/transport/netty/NettyTransportChannel.java

```diff
@@ -134,6 +141,14 @@ public void sendResponse(Throwable error) throws IOException {
         transportServiceAdapter.onResponseSent(requestId, action, error);
     }
 
+    private void close() {
```

Comment:
I still think that `close()` is fine as name but I am ok if you want to change it.